### PR TITLE
Add schedule for all days

### PR DESCRIPTION
**Pull request by afwlehmann**
The final (?) schedule corresponding to what we worked out until today. Please check everything in detail.

### CODE REVIEW

File: templates/schedule.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
+<h2>Schedule</h2>
+
+<p>Please note that all times are CST.</p>
+
+<table style="border-spacing: 2px; border-collapse: separate">
```
Comment (fmthoma) [inline templates/schedule.html:5]: Add fixed column widths and column borders to make table more readable

File: templates/schedule.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    <td colspan=2>Keynote</td>
+  </tr>
+  <tr>
+    <td align="right">12:30</td>
+    <td colspan=2>Lunch</td>
+  </tr>
+  <tr>
+    <td align="right">14:00</td>
+    <td>Hack on</td>
+    <td>Datatype Generic Programming</td>
```
Comment (fmthoma) [inline templates/schedule.html]: Hyphen? (»Datatype-Generic«)

Comment (afwlehmann) [reply templates/schedule.html]: Will do.

File: templates/schedule.html (hunk 1 above)
Comment (afwlehmann) [reply templates/schedule.html:5]: I tried column bordes in a first attempt but found that they made the table less readable. About the column widths: Do you have something specific in mind or do you want towards having more space between adjacent cells?

File: templates/schedule.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    <td align="right">11:00</td>
+    <td colspan=2>Keynote</td>
+  </tr>
+  <tr>
+    <td align="right">14:00</td>
+    <td colspan=2>Coffee &amp; Cake</td>
+  </tr>
+  <tr>
+    <td align="right">14:30</td>
+    <td colspan=2>Time for everybody to present their results (if you will)</td>
```
Comment (kosmikus) [inline templates/schedule.html]: Sounds very strange to me. Why not simply `Project presentations`?

Comment (afwlehmann) [reply templates/schedule.html]: Yeah, why not. I'll change it.

File: templates/schedule.html (hunk 1 above)
Comment (fmthoma) [reply templates/schedule.html:5]: My main intention is to make the difference between all-track timeslots (like breakfast) and track-specific ones (like »hack on«/»workshop«) more obvious.

Comment (afwlehmann) [reply templates/schedule.html:5]: As discussed via mail.

**Pull request merged**